README.md

```diff
@@ -42,15 +42,16 @@ If you would like to see your name added, let us know on Slack, or send a PR ple
 
 ## <a name="getting-started"></a>Getting Started
 
+Ensure your computer is behind a firewall that blocks port 4040 then,
```

Comment:
Can we add a small note like, **If you're using Scope in Kubernetes then do not expose it as a NodePort service** ? 

Also, it'd be good to have these instructions in **bold** format. :smile:

Reply:
done